src/lesson5/inheritance/Mission.java

```diff
@@ -0,0 +1,19 @@
+package lesson5.inheritance;
+
+public class Mission extends SpaseX{
+    public boolean isHumanCrew = false;
+
+    public Mission(String mission, String captain, String launchSite, boolean isHumanCrew){
+        super(mission, captain, launchSite);
+        this.isHumanCrew = isHumanCrew;
+    }
+
+    public void isCrew(){
+        if(isHumanCrew = false){
```

Comment:
I detect that this code is problematic. According to the [Correctness (CORRECTNESS)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#correctness-correctness), [QBA: Method assigns boolean literal in boolean expression (QBA_QUESTIONABLE_BOOLEAN_ASSIGNMENT)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#qba-method-assigns-boolean-literal-in-boolean-expression-qba-questionable-boolean-assignment).
This method assigns a literal boolean value (true or false) to a boolean variable inside an if or while expression. Most probably this was supposed to be a boolean comparison using ==, not an assignment using =.